src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a ^ c) && (b ^ d)) || ((a ^ d) && (b ^ c));
```

Comment:
```suggestion
        return (a ^ c) && (b ^ d) || (a ^ d) && (b ^ c);
```
avoid redundant braces please